api/src/main/java/org/openmrs/obs/ComplexData.java

```diff
@@ -39,7 +41,7 @@
 	 * @param title Name or brief description of ComplexData.
 	 * @param data The complex data for an Obs
 	 */
-	public ComplexData(String title, Object data) {
+	public ComplexData(String title, DataSource data) {
```

Comment:
hi @dkayiwa  @mseaton  @ibacher  @mogoodrich  could this be the correct way of introducing support for streams under ComplexData